src/CodeGenerator/src/Generator/GeneratorHelper.php

```diff
@@ -70,6 +70,8 @@ public static function parseDocumentation(string $documentation, bool $short = t
             '<p>' => '',
             '<p/>' => '',
             '</p>' => "\n",
+            '<u>' => '',
+            '</u>' => '',
```

Comment:
I honestly did not know about this HTML tag. It was used in a comment here: https://github.com/async-aws/aws/pull/1087/files#diff-393723a3db6ca45a459be2595a3c6fb8195c8172fe11b9200f1bc91246b52beaR8

Reply:
good catch